installers/ansible/inventory

```diff
@@ -26,6 +30,14 @@ crunchy_debug='false'
 # ==================
 # kubernetes_context=''
 
+# Deploy with PGO-Installer
+# ==================
+# Note: Enable kubernetes_in_cluster if you are using
+# the pgo-installer image to install the operator
+# ==================
+# kubernetes_in_cluster=false
```

Comment:
This name is a bit confusing. Why not

`use_pgo_installer`

Reply:
This variable existed before my changes so i didn't name it. AIUI it disables some of the checks that are in place when installing with ansible locally.  It might make sense to have a `use_pgo_installer` that will go in and enable this option. Or we could rename it but that might be something to run by chris since he created it.

Reply:
`kubernetes_in_cluster=true` effectively disables the checks and actions we take around `kubernetes_context` and `openshift_host`. It indicates that you _want_ to use `kubectl` as-is. In the GCP installer there is _no_ kubeconfig file at all. There is no correct setting for context.